package.json

```diff
@@ -41,11 +41,12 @@
     "grunt": "^0.4.5",
     "grunt-shell": "^1.1.1",
     "gruntfile-gtx": "^0.3.0",
+    "lodash": "~3.8.0",
```

Comment:
just a preference and possibly a performance thing... should we include all of lodash or just the packages we require from it? I'm in favour of the latter...